src/api/database/find.rst

```diff
@@ -987,12 +984,18 @@ built using MapReduce Views.
     :code 200: Index created successfully or already exists
     :code 400: Invalid request
     :code 401: Admin permission required
+    :code 404: Database not found
     :code 500: Execution error
 
 **Index object format for JSON type indexes**
 
-The index object is a JSON array of field names following the :ref:`sort
-syntax <find/sort>`. Nested fields are also allowed, e.g. `"person.name"`.
+The index object is a JSON object with the following fields:
+
+-  ``fields``: array of field names following the :ref:`sort
```

Comment:
This section should be formatted similar to this: 

https://github.com/apache/couchdb-documentation/blame/main/src/api/server/common.rst#L644-L658

Use the same `:json ...` type structure so that all of the fields render correctly.

Reply:
I've updated the PR. Here is a screenshot of what the new part looks like:

![image](https://user-images.githubusercontent.com/614261/100710804-9612b080-33b0-11eb-8eae-c335b938d4ac.png)

Let me know if this is not the way it's supposed to be :)